clusterloader2/README.md

```diff
@@ -85,10 +85,11 @@ export CL2_ACCESS_TOKENS_QPS=5
 ## Measurement
 
 Currently available measurements are:
-- **APIResponsiveness** \
-This measurement creates summary for latency and number for server api calls.
+- **APIResponsivenessPrometheusSimple** \
+This measurement creates summary for latency and number for server api calls based on the data collected by the prometheus-operator/prometheus server. 
```

Comment:
Sorry - it seems I wasn't clear again.

I was simply suggesting to remove "prometheus-operator" and just leave prometheus server.
See lines 94-98 - those two should be consistent.

Reply:
so what is the difference between APIResponsivenessPrometheusSimple and  APIResponsivenessPrometheus? if the description is same, why we need both of them?

Reply:
The readme is misleading. 

The simple simple gets percentiles over the whole test run.
The non-simple - implements the window-based official definition: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md

Reply:
thanks for your information. just updated and added percentiles to APIResponsivenessPrometheusSimple